api-compact.yaml

```diff
@@ -10,6 +10,9 @@
 #   /enterprise/api/v1/catalogs/
 #   /enterprise/api/v1/catalogs/{id}/
 #   /enterprise/api/v1/catalogs/{id}/courses/
+#   /enterprise/api/v1/enterprise_catalogs/
+#   /enterprise/api/v1/enterprise_catalogs/{uuid}/
```

Comment:
@UmanShahzad We still have underscores here. Should probably turn them into dashes after fbdf6222b8925de0ed2b4d2d302d014fb03fbdd4.